### DISCUSSION TITLE
No folders on Share partition

### DISCUSSION THREAD

**Discussion by thahare** [rg35xx-cfw/rg35xx-cfw.github.io#112] [Q&A] [answered]
Hello,

I tried installing 3 times already but I do not have folders on the Share partition. I followed TechDweep's installation youtube video. Did I do something wrong or is this intended behavior? I am using the 64 Gb card that cam with the device and using the latest version of the installer (20240208). Many thanks in advance for your help.

**Comment by acmeplus** [comment, marked as answer]
You need to flash the image first, then boot with that card and wait until it boots completely (you will see emulationstation running). After that, you can take that card out and insert it on your computer and you will see the SHARE partition.

Note that if you have another card on the second TF2 port, that card will be used instead, so no share is created in the first card.

**Reply by thahare** [reply]
I had a card in the second slot. That's why it did not it did not show. Thanks for your answer.
